codalab_service.py

```diff
@@ -716,11 +718,13 @@ def build_image(self, image):
         else:
             build_args = ''
 
+        # We need the frontend env variables during build time
         if image == 'frontend' and not self.args.dev:
-            CODALAB_RECAPTCHA_SITE_KEY = os.environ.get('CODALAB_RECAPTCHA_SITE_KEY')
-            build_args += '--build-arg REACT_APP_CODALAB_RECAPTCHA_SITE_KEY={}'.format(
-                CODALAB_RECAPTCHA_SITE_KEY
-            )
+            with open('frontend/.env.production', "w") as f:
```

Comment:
Need w+ so it will overwrite an existing env file. Also, let's just call it `.env` for simplicity.

```suggestion
            with open('frontend/.env', "w+") as f:
```

Reply:
Fixed. Thanks:)